modules/ossm-rn-known-issues.adoc

```diff
@@ -32,39 +32,32 @@ These are the known issues in {ProductName}:
 
 * link:https://github.com/istio/istio/issues/14743[Istio-14743] Due to limitations in the version of Istio that this release of {ProductName} is based on, there are several applications that are currently incompatible with {ProductShortName}. See the linked community issue for details.
 
+* https://issues.redhat.com/browse/OSSM-882[OSSM-882] Namespace is in the accessible-namespace list but does not appear in Kiali UI. By default, Kiali will not show any namespaces that start with "kube" because these namespaces are typically internal-use only and not part of a mesh.
```

Comment:
```suggestion
* https://issues.redhat.com/browse/OSSM-882[OSSM-882] Namespace is in the accessible_namespaces list but does not appear in Kiali UI. By default, Kiali will not show any namespaces that start with "kube" because these namespaces are typically internal-use only and not part of a mesh.
```